### DISCUSSION TITLE
Change device type from router to endpoint

### DISCUSSION THREAD

**Discussion by stefanchiriac** [Koenkk/zigbee2mqtt#14185] [General]
Hi,
I have a couple of zigbee light bulbs that also act as routers. The problem is that they don't stay on all the time and each time I close the light they mess up the network.
There is a way to change the device type from router to endpoint, or to disable the router capabilities?

**Comment by Koenkk** [comment]
> There is a way to change the device type from router to endpoint, or to disable the router capabilities?
 
This is not possible

**Reply by arozhankov** [reply]
I have the same problem. This function is very necessary. I understand that this is a function of the devices, not the coordinator and not Z2Q. But I see the mistake of the Zigbee developers here.

**Reply by GuillaumeBraillon** [reply]
Hello,
What about this ? https://github.com/Koenkk/zigbee2mqtt/issues/2380
This seems to work, if it is confirmed it might be possible to add an option in ZigBee2mqtt?
